tests/test_freqtradebot.py

```diff
@@ -2691,16 +2709,16 @@ def test_execute_trade_exit_custom_exit_price(default_conf, ticker, fee, ticker_
         'trade_id': 1,
         'type': RPCMessageType.SELL,
         'exchange': 'Binance',
-        'pair': 'ETH/BTC',
+        'pair': 'ETH/USDT',
         'gain': 'profit',
-        'limit': 1.170e-05,
-        'amount': 91.07468123,
+        'limit': 2.25,
```

Comment:
pdb is telling me `{'limit': 2.244}` and I'm confused why it's not `2.25` considering that's the value that I set in `    freqtrade.strategy.custom_exit_price = lambda **kwargs: 2.25`